packages/block-editor/src/components/block-controls/groups.js

```diff
@@ -8,13 +8,15 @@ const BlockControlsBlock = createSlotFill( 'BlockControlsBlock' );
 const BlockControlsInline = createSlotFill( 'BlockFormatControls' );
 const BlockControlsOther = createSlotFill( 'BlockControlsOther' );
 const BlockControlsParent = createSlotFill( 'BlockControlsParent' );
+const BlockControlsAlt = createSlotFill( 'BlockControlsParent' );
```

Comment:
```suggestion
const BlockControlsAlt = createSlotFill( 'BlockControlsAlt' );
```

I am looking forward to this improvement. Feel free to ping me when PR is ready for testing.